File: app/src/main/java/hu/pe/biko/biko/FinishedDialog.java
package hu.pe.biko.biko;

import android.app.Dialog;
import android.app.DialogFragment;
import android.os.Bundle;
import android.support.v4.app.FragmentManager;
import android.support.v7.app.AlertDialog;
import android.view.LayoutInflater;
import android.view.View;

/**
 * Created by nikita on 28.05.17.
 */

public class FinishedDialog extends DialogFragment {

    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
        LayoutInflater inflater = getActivity().getLayoutInflater();
        View view = inflater.inflate(R.layout.finished_dialog, null);
        builder.setView(view);
        // Остальной код
        return builder.create();
    }
}

File: app/src/main/java/hu/pe/biko/biko/FinishedDialog.kt
package hu.pe.biko.biko

import android.app.Dialog
import android.app.DialogFragment
import android.os.Bundle
import android.support.v4.app.FragmentManager
import android.support.v7.app.AlertDialog
import android.view.LayoutInflater
import android.view.View

/**
 * Created by nikita on 28.05.17.
 */

class FinishedDialog : DialogFragment() {

    override fun onCreateDialog(savedInstanceState: Bundle): Dialog {
        val builder = AlertDialog.Builder(activity)
        val inflater = activity.layoutInflater
        val view = inflater.inflate(R.layout.finished_dialog, null)
        builder.setView(view)
        // Остальной код
        return builder.create()
    }
}
